src/FixerDefinition/FixerDefinition.php

```diff
@@ -36,9 +36,9 @@ public function __construct(
         $summary,
         $description,
         array $codeSamples,
-        $configurationDescription,
+        $configurationDescription = null,
         array $defaultConfiguration = null,
-        $riskyDescription
+        $riskyDescription = null
```

Comment:
I'm not saying that $riskyDescription is not nullable.
Let think in PHP7.1 world.
`$riskyDescription` may be `string` or `null`.
So the type in prototype is `?string $riskyDescription`.
And that typehinting is already in method phpdoc.

What I'm saying that I don't like to have default value here.
`?string $riskyDescription` is good for me
`?string $riskyDescription = null` is not